templates/vue/cypress/integration/lightbox/activity.spec.js

```diff
@@ -35,23 +98,125 @@ describe("Activity", () => {
 
       cy.tick(3600 * 1000) // move forward by 1 hour
       cy.contains("1:00:05").should("be.visible")
+    })
+  })
+
+  it("should be able to edit questions they have already answered by choosing multi-line text question", () => {
```

Comment:
Again, this sentence is hard to understand. Are you meaning:
```suggestion
  it("should be able to edit multi-line text already answered questions", () => {
```
?

Reply:
@wynnset I will change it to should be able to edit single-line text already answered question, since I answered a single-line text question then sets it to be multi-line

Reply:
Again, I think use cases where an author is answering their own activity question is not one we need to worry about. This would be more useful if you log in as a different user.